Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [workspace]
-default-members = [
-    "relay"
-]
-members = [
-    "methods",
-    "relay"
-]
+members = ["methods", "relay"]
+
+[workspace.dependencies]
```

Comment:
If we pull in `risc0` as a submodule can't we use the path= imports here for all of these?